.gitignore

```diff
@@ -11,6 +11,8 @@ target
 # Bazel
 bazel-bin
 bazel-examples
+bazel-examples-alts
```

Comment:
No, let's not create a separate workspace for each set of examples.

Reply:
Done.